capstone/src/main/webapp/sheetsScript.js

```diff
@@ -0,0 +1,90 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// API key from the Developer Console
+const API_KEY = 'SHEETS_API_KEY'; // TODO: Create Java servlet to return key
+
+// Array of API discovery doc URLs for APIs
+const DISCOVERY_DOCS = ["https://sheets.googleapis.com/$discovery/rest?version=v4"];
+
+// Authorization scopes required by the API; multiple scopes can be
+// included, separated by spaces.
+const SCOPES = "https://www.googleapis.com/auth/spreadsheets";
+
+/** Collects the user's input and submits it for analysis */
+async function gatherSheetsInput() {
```

Comment:
Just FYI, it seems like this `gatherSheetsInput` is very similar to `gatherDocsInput` from #23.  Possibly in a follow up CL you might want to refactor them so they use more similar code (possibly making some shared functions, or even making classes that inherit/implement the same interface)

Reply:
Noted - I'll see if I can do this